annotation-web/src/main/java/eu/europeana/annotation/web/service/controller/jsonld/WebAnnotationAuxiliaryMethodsRest.java

```diff
@@ -66,10 +66,9 @@ public ResponseEntity<String> createAnnotationByTypeJsonld(
 	return storeAnnotation(motivation, indexOnCreate, annotation, authentication);
     }
 
-    @RequestMapping(value = "/annotations/deleted", method = RequestMethod.GET, produces = {
-	    HttpHeaders.CONTENT_TYPE_JSON_UTF8 })
+    @RequestMapping(value = "/annotations/deleted", method = RequestMethod.GET)
```

Comment:
the produces are needed for content negotiation mechanism and for filling the content type response header

Reply:
Ok.